index.html

```diff
@@ -18,13 +18,22 @@
         <div id="container-logo"><a id="myLogo" >My Logo</a></div>  
         <nav id="navbar"> 
             <ul>
-                <li><a>Portfolio</a></li>
-                <li><a>About</a></li>
-                <li><a>Contact</a></li>
+                <li><a href="#headline">Portfolio</a></li>
+                <li><a href="#myself-section">About</a></li>
+                <li><a href="#contact">Contact</a></li>
             </ul>    
-            <a id="menu-action"><img id="menu" src="css/img/hamburger.svg"></a>
+            <button id="menu-action"><img id="menu" src="css/img/hamburger.svg"></button>
         </nav>
     </header>
+    <div id="menu-mobile" class="menu-mobile">
+        <button type="button" class="close-menu">&times;</button>
+        <ul>
+            <li><a id="headline-link" href="#headline">Portfolio</a></li>
+            <li><a id="myself-link" href="#myself-section">About</a></li>
+            <li><a id="contact-link" href="#contact">Contact</a></li>
+        </ul>    
+    </div>

```

Comment:
- [ ] You have done a great job 💪 implementing the mobile📱 menu, but there is still a difference between the Figma design and your implementation. If you look at the Figma design, there is a blur effect on the content in the background of the mobile menu. 🙏Please ensure that you include the blur effect, as it will make your design look as cool as the Figma design.

Your design:
![image](https://github.com/BiancaMe/Portfolio-Mobile/assets/63445083/1414fc19-29b0-4ebc-960a-1c0164714712)

Figma design:
![image](https://github.com/BiancaMe/Portfolio-Mobile/assets/63445083/a9761b45-fd2b-4f07-9024-75ba856d1b48)